src/components/AddFile.tsx

```diff
@@ -122,26 +125,26 @@ const AddFile = ({
           ) : (
             <>
               {confirm ? (
-                <div className="flex-row self-center h-[250px] max-w-[35%] flex-col place-content-center gap-y-10 rounded-lg bg-white p-10 text-center text-lg">
+                <div className="h-[250px] max-w-[35%] flex-row flex-col place-content-center gap-y-10 self-center rounded-lg bg-white p-10 text-center text-lg">
```

Comment:
Noting that this styling has conflicts between `flex-row` and `flex-col`. You guys don't have to fix the confirmation now, though!